control-plane/agents/src/bin/core/controller/reconciler/nexus/mod.rs

```diff
@@ -276,6 +279,21 @@ pub(super) async fn missing_children_remover(
     result
 }
 
+/// Recreate the given nexus on its associated node.
+pub(crate) async fn nexus_recreate(
+    registry: &Registry,
+    nexus: &mut OperationGuardArc<NexusSpec>,
+) -> Result<(), SvcError> {
+    let poll_context = PollContext::from(
+        &PollEvent::Triggered(PollTriggerEvent::VolumeRepublish),
+        registry,
+    );
+    match missing_nexus_recreate(nexus, &poll_context).await {
+        Ok(_) => Ok(()),
+        Err(error) => Err(error),
+    }
```

Comment:
nit: can simply if you prefer it:
```suggestion
    missing_nexus_recreate(nexus, &poll_context).await?;
    Ok(())
```

Reply:
Yep.